IFC4x3/Properties/g/GroundResistance_2NdoXdKT98XeEMmk8dSBQF/Documentation.md

```diff
@@ -0,0 +1 @@
+The soil or ground resistance to electrical current from the cable fitting
```

Comment:
I still do not understand this. The cable fitting induces resistance in the ground/soil?
```suggestion
The soil or ground resistance to electrical current from the cable fitting.
```